.github/workflows/ci.yml

```
@@ -44,6 +44,7 @@ jobs:
       - uses: julia-actions/julia-runtest@v1
         env:
           JULIA_DISTRIBUTED_TESTING_STANDALONE: 1
+          JULIA_NUM_THREADS: 4
```

```suggestion
          JULIA_NUM_THREADS: 4,4
```
Shake out having lots of threads in both pools?

---

Yep makes sense, fixed in 2874281.